packages/next/types/react-dom.d.ts

```diff
@@ -27,9 +27,7 @@ declare module 'react-dom/server.edge' {
   export function resume(
     children: JSX.Element,
     postponedState: object,
-    options?: {
-      onError?: (error: Error, errorInfo: unknown) => void
-    }
+    options?: ResumeOptions
```

Comment:
we had these types already we just weren't actually using them here until now. I assume this was an overight